docs/source/features/sheet_management/search.rst

```diff
@@ -0,0 +1,111 @@
+..
+ Copyright 2022 Graviti. Licensed under MIT License.
+
+##################
+ Search Operation
+##################
+
+This topic describes Graviti DataFrame search operations:
+
+   * :py:meth:`~graviti.dataframe.frame.query``
+   * :py:meth:`~graviti.dataframe.frame.appaly``
+
+Now only supports calling graviti OpenAPI to do the search.
```

Comment:
what means? sdk can do the search in local without OpenAPI in the future?

Reply:
right

Reply:
Can we put it another way, such as the current search is not done locally. So if the amount of data is particularly large, it may be slow, please be patient.